### DISCUSSION TITLE
Welcome to pyvista Discussions!

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#1393] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

We're still using [slack](http://slack.pyvista.org/) as a primary channel of communication, but this is another channel for posting general comments rather than in the [Issues](https://github.com/pyvista/pyvista/issues).  If we don't use it, we can shut it down, but I think it's a nice place to have discussions and post general questions/answers.

One other idea is to move `pyvista/support` here (or at least redirect questions here) so we can have one main location for those questions/answers since they're less issues and more usage questions.

**Comment by tkoyama010** [comment]
+100 for using GitHub discussion. I made PR to add discussion URL to issue template and move pyvista-support url to discussion https://github.com/pyvista/pyvista/pull/1395 .

**Reply by akaszynski** [reply]
People are posting here as well and it seems a bit more friendly and is a good place for general or basic discussions.

https://github.com/pyvista/pyvista/discussions/1465
